docs/source/contrib/tutorials/make_rst/creating_rst/creating-rst.rst

```diff
@@ -0,0 +1,429 @@
+Creating rST Documents
+======================
+
+.. note::
+   It is assumed that you have created a GitHub account and been given the proper 
+   permissions to work within this repository for training. If you can read this,
+   you probably have, but double-check with your friendly FTC Support person.
```

Comment:
The goal of ftcdocs is to allow any and all to contribute without any permissions being granted. The final version will reflect this goal. This is another artifact because of this documentations origins as internal documentation.